Jobs/MM_job.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+"""Run Singlepoint Calculations"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-09-19'
+
+#   // IMPORTS //
```

Comment:
These comments seem a bit unnecessary. I mean "import something" pretty much speaks for itself.